18-minimal/Dockerfile.c8s

```diff
@@ -0,0 +1,68 @@
+FROM ubi8/ubi-minimal:latest
+
+EXPOSE 8080
+
+# Add $HOME/node_modules/.bin to the $PATH, allowing user to make npm scripts
+# available on the CLI without using npm's --global installation mode
+# This image will be initialized with "npm run $NPM_RUN"
+# See https://docs.npmjs.com/misc/scripts, and your repo's package.json
+# file for possible values of NPM_RUN
+# Description
+# Environment:
+# * $NPM_RUN - Select an alternate / custom runtime mode, defined in your package.json files' scripts section (default: npm run "start").
+# Expose ports:
+# * 8080 - Unprivileged port used by nodejs application
+ENV APP_ROOT=/opt/app-root \
```

Comment:
This is not needed. Variable APP_ROOT is defined here: https://github.com/sclorg/s2i-base-container/blob/master/core/Dockerfile.c8s#L25

Reply:
But for minimal, we use `FROM ubi8/ubi-minimal:latest` so we need to define what is usually defined in the core image.